src/Model/MyParcelCustomsItem.php

```diff
@@ -157,10 +157,16 @@ public function getClassification()
      * @link https://www.cbs.nl/nl-nl/deelnemers-enquetes/deelnemers-enquetes/bedrijven/onderzoek/lopend/internationale-handel-in-goederen/idep-codelijsten
      *
      * @param int $classification
+     *
      * @return $this
+     * @throws \MyParcelNL\Sdk\src\Exception\MissingFieldException
      */
-    public function setClassification($classification)
+    public function setClassification(int $classification): self
     {
+        if (!$classification) {
```

Comment:
$classification kan alleen een int zijn, dus dan moet je checken met:
`if (0 !== $classification) {`

of wel een null toestaan

Reply:
`if (0 !== $classification) {` zal niet gaan werken omdat hij dan altijd in de if statement komt zodra er een HS code is ingevoerd.

Dan zal je fit moeten doen : `if (0 === $classification) {`.

Alleen zal dit geen oplossing zijn omdat je NULL terug krijgt als er geen HS code is.
Hierdoor heb ik null moeten toestaan.